pkg/reconciler/sequence/controller.go

```diff
@@ -61,5 +66,16 @@ func NewController(
 		Handler:    controller.HandleAll(impl.EnqueueControllerOf),
 	})
 
+	sequenceGK := flowsv1.SchemeGroupVersion.WithKind("Sequence").GroupKind()
+
+	// Enqueue the Sequence, if we have an EventPolicy which was referencing
+	// or got updated and now is referencing the Sequence
+
```

Comment:
```suggestion
	// or got updated and now is referencing the Sequence
```
nit to remove the empty line